src/parameterized/studentt.jl

```diff
@@ -1,17 +1,20 @@
 
 # StudentT distribution
 
-export StudentT
+export StudentT, HalfStudentT
 
 @parameterized StudentT(ν) ≪ Lebesgue(ℝ)
 
 @kwstruct StudentT(ν)
+@kwstruct StudentT(ν,μ)
+@kwstruct StudentT(ν,σ)
+@kwstruct StudentT(ν,μ,σ)
 
-StudentT(nt::NamedTuple{(:ν,:μ,:σ)}) = Affine(nt, StudentT())
-StudentT(nt::NamedTuple{(:ν,:μ,:ω)}) = Affine(nt, StudentT())
-StudentT(nt::NamedTuple{(:ν,:σ,)}) = Affine(nt, StudentT())
-StudentT(nt::NamedTuple{(:ν,:ω,)}) = Affine(nt, StudentT())
-StudentT(nt::NamedTuple{(:ν,:μ,)}) = Affine(nt, StudentT())
+StudentT(nt::NamedTuple{(:ν,:μ,:σ)}) = Affine(NamedTuple{(:μ,:σ)}(nt), StudentT(ν=nt.ν))
```

Comment:
I know this isn't part of this PR, but overloading constructors to construct something else can cause problems, the simplest being that something like `StudentT(0, 1, 30) isa StudentT` is `false`.

Reply:
Yeah, I agree that's a little weird. But the upside is pretty huge. This approach allows us to fine-tune code for `Affine`, and have that same code work for multivariate-anything. So implementing MvNormal gives us MvStudentT etc for free.

To me it seems worth it, but if you disagree we should talk through it

Reply:
Made this an issue:
https://github.com/cscherrer/MeasureTheory.jl/issues/157

I'll resolve this now, we can discuss there